### ISSUE TITLE
V2 followup

### ISSUE THREAD

**Issue by xmo-odoo** [odoo/runbot#890]
Grab-bag of smaller improvements or tuneups to perform in the wake of the major update to commands handling & dashboards:

- [x] only notify the fw heads aka the forward-most non-detached PR of each batch / bundle thing, this requires taking the set of descendants then removing all the attached ancestors of any given PR
- [x] if an "empty commit" merge error occurs  odoo/odoo#165777
    ```
    Traceback (most recent call last):
      File "runbot_merge/models/stagings_create.py", line 328, in stage_batch
        method, new_heads[pr] = stage(pr, info, related_prs=(batch.prs - pr))
      File "runbot_merge/models/stagings_create.py", line 448, in stage
        new_head = fn(pr, info, pr_commits, related_prs=related_prs)
      File "runbot_merge/models/stagings_create.py", line 506, in stage_rebase_ff
        head, mapping = info.repo.rebase(info.head, commits=commits)
      File "runbot/runbot_merge/git.py", line 156, in rebase
        raise MergeError(
    odoo.addons.runbot_merge.github.MergeError: commit XXX results in an empty tree when merged, it is likely a duplicate of a merged commit, rebase and remove.
    
    The above exception was the direct cause of the following exception:
    
    Traceback (most recent call last):
      File "runbot_merge/models/stagings_create.py", line 259, in stage_batches
        staged |= stage_batch(env, batch, staging_state)
      File "runbot_merge/models/stagings_create.py", line 335, in stage_batch
        raise exceptions.MergeError(pr) from e
    odoo.addons.runbot_merge.exceptions.MergeError: runbot_merge.pull_requests(xxx)
    ```
    then a message is sent on the PR, but the PR was not put in error directly, only after >200 attempts was it finally put out

    might be an off by one of some sort: the PR kept being staged when it was the first stageable, and it stopped and was finally put in error when it was the *second*

- [x] the dashboard thing seems to have issues when a limit is set: "Expected singleton: runbot_merge.branch(1, 81)" visible at odoo/odoo#168218

    the issue is almost certainly with `limit = genealogy.prs.limit_id.sorted(lambda b: (b.sequence, b.name))` or specifically the `limit_high` line which assumes at most one record but that's what limit *should* be, the high limit, this is missing a slice.

    Also: make sure that when creating a forward port the limit is either copied from the original or unset, on odoo/odoo#165777 the limit of the forward port was set to master even as the original was 17.3, not sure why, might be worth checking older open ports where the source had a limit...

   Also: move `limit` to forward-port tab, and add domain so the branch is in the same project as the PR's repo
- [x] alias "ignore" to "fw=no", deprecate "ignore" (send a warning message): "robodoo ignore" is a lot less clear and obvious than "fw-bot ignore" was, even with the feedback comment
- [x] check that ready/staged was added to the caching keys
- [x] ?replace "checked" by something else for statuses? -> replace by r+ and CI, add "not" before?, also increase font size a touch

I *think* the notifications spam was just because I restarted the mergebot in the sending of it checking for outdated forward ports. I've updated the reminder cron to run at 4AM for now.
